docs/spec/src/integration/spec/7-op-secure-integration-workflow.md

```diff
@@ -0,0 +1,112 @@
+# EigenDA OP Secure integration
+
+This document presents an overview on how EigenDAA plugs into Optimism (OP) Stack.
+- `write` and `read` path in a L2 rollup
+- Why the `read` path must stay live (even with a misbehaving op-batcher)
+- Adding an EigenDA stage to the OP derivation pipeline
+- Hokulea, Rust library that defines and implement the Eigenda derivation rule
+- How Hokulea works in both interactive fault-proof VMs and zkVMs
+
+## Write and Read path in L2 consensus
+
+A rollup system can be splitted into two parts: write path to L1 and read path from L1
+
+| Path      | Direction | Purpose                                    | Main actor                   |
+| --------- | --------- | ------------------------------------------ | ---------------------------- |
+| **Write**| L2 → L1   | Liveness – keep blocks flowing             | `op-batcher` + EigenDA proxy |
+| **Write**| Direct on L1   | Censorship resistance + Deposit        | Rollup users        + Opimism Portal     |
+| **Read**  | L1 → L2   | Safety – all nodes see the same block list | OP derivation pipeline       |
+
+
+- The `write path` ensures the liveness of the L2 consensus. It consists of L2 blocks produced by op-batch and L1 induced transactions.
+- The `read path` controls the safety of the L2 consensus. It ensures all L2 consensus node sees an identical list of L2 blocks, then a EVM can produce identical L2 state
```

Comment:
```suggestion
- The `read path` controls the safety of the L2 consensus. It ensures that all L2 consensus nodes see an identical list of L2 blocks, such that an EVM can produce identical L2 state
```

Reply:
this is not 100% accurate since DP produces payload attributes and the EVM produces the blocks. but maybe not worth going into that level of details